tools/linter/codespell/matcher.json

```diff
@@ -0,0 +1,26 @@
+{
```

Comment:
seems this file make CI fail.

Reply:
The failure is because `lint.mk` said
```
	    printf '::add-matcher::$(CURDIR)/tools/codespell/matcher.json\n'; \
```
Which is where it was pre-rebase. But meanwhile, the directory had been moved to `./tools/linter/codespell/`, but I didn't catch that when doing the rebase.

It should be fixed now.